packages/beacon-node/src/metrics/metrics/lodestar.ts

```diff
@@ -1373,6 +1389,21 @@ export function createLodestarMetrics(
       help: "regen function total errors",
       labelNames: ["entrypoint", "caller"],
     }),
+    regenFnAddPubkeyTime: register.histogram({
+      name: "lodestar_regen_fn_add_pubkey_time_seconds",
+      help: "Historgram of time spent on adding pubkeys to all state cache items in seconds",
+      buckets: [0.01, 0.1, 0.5, 1, 2, 5],
+    }),
+    regenFnDeletePubkeyTime: register.histogram({
+      name: "lodestar_regen_fn_delete_pubkey_time_seconds",
+      help: "Histrogram of time spent on deleting pubkeys from all state cache items in seconds",
+      buckets: [0.01, 0.1, 0.5, 1, 2, 5],
```

Comment:
I don't think 2 or 5 seconds will be useful here (but may be worth investigating)